libr/core/cbin.c

```diff
@@ -2759,7 +2759,7 @@ static int bin_sections(RCore *r, int mode, ut64 laddr, int va, ut64 at, const c
 				str = r_str_newf ("[%02d] %s %s size %" PFMT64d" named %s%s%s",
 				                  i, perms, type, size,
 				                  pfx? pfx: "", pfx? ".": "", section->name);
-				r_meta_add (r->anal, R_META_TYPE_COMMENT, addr, addr, str);
+				r_meta_set (r->anal, R_META_TYPE_COMMENT, addr, 1, str);
```

Comment:
Why 1 and not size?

Reply:
I think comments always have size 1, that's also how it has been before. Or well `addr, addr` is size 0 to be exact.